app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java

```diff
@@ -148,8 +149,20 @@ public void onBindViewHolder(@NonNull ViewFilesHolder holder, int position) {
         final int filePosition = position;
         String[] name = fileName.split("/");
 
+        File file = new File(fileName);
+        Date lastModDate = new Date(file.lastModified());
+
+        String[] formatdate = lastModDate.toString().split(" ");
+        String time = formatdate[3];
+
+        String[] formattime =  time.split(":");
```

Comment:
Add a new function in fileutils to get the formatted date. And move all this code there